openmicroscopy/openmicroscopy
# Group user menu speed up

This PR speeds up loading of group user menu in Web. Previously on each group there were additional calls loading user map. Now it is improved and entire list is loaded at once.

To test:
 - click on group user many and see in Firebug how fast it loads

cc: @pwalczysko 

Petr On trout now 2:30 min should be reduced to 2.40 sec. I hope that will help
